dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/AccessToken.java

```diff
@@ -105,4 +103,45 @@ public Date getUpdateTime() {
     public void setUpdateTime(Date updateTime) {
         this.updateTime = updateTime;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Think about the real scenario ：
Id of accessToken is generated by MySQL, so id is unique.  So what is the best implementation of equals method ？